tools/exporter/tests/create_unit_test.py

```diff
@@ -1,68 +0,0 @@
-import sys
```

Comment:
Code is messy. It does not have comments or error handling. If a customer decides to use it, I think it would cause more questions and confusion than value.